addons/editor-devtools/DevTools.js

```diff
@@ -1505,8 +1505,15 @@ export default class DevTools {
     }
 
     if (this.floatInp && !e.target.closest("#s3devIDDOut")) {
-      // If we click outside the dropdown, then instigate the hide code...
-      this.hideFloatDropDown();
+      if (
+        !e.shiftKey ||
+        (!document.getElementsByClassName("injectionDiv")[0].contains(e.target) &&
+          !e.target.classList.contains("blocklyHtmlInput")) ||
+        e.target.matches(".blocklyFlyoutButton, .blocklyFlyoutButton *, .blocklyTouchTargetBackground")
```

Comment:
Could you add a code comment for this? That selector is scary lol

Reply:
Done, along with other parts of that condition